src/containerapp/azext_containerapp/_help.py

```diff
@@ -2113,27 +2113,27 @@
       text: |
           az containerapp sessionpool create -n mysessionpool -g MyResourceGroup \\
               --container-type CustomContainer --environment MyEnvironment \\
-              --cpu 0.5 --memory 1Gi --target-port 80 --image MyImage \\
+              --cpu 0.5 --memory 1Gi --target-port 80 --image myregistry.azurecr.io/MyImage:Tag \\
```

Comment:
The `--image` supports not only azure.io image. I think it doesn't need to update the docs.